engine/src/flutter/impeller/entity/contents/content_context.cc

```diff
@@ -329,6 +329,7 @@ ContentContext::ContentContext(
     solid_fill_pipelines_.CreateDefault(*context_, options);
     texture_pipelines_.CreateDefault(*context_, options);
     fast_gradient_pipelines_.CreateDefault(*context_, options);
+    line_pipelines_.CreateDefault(*context_, options);
```

Comment:
We could put this behind the flag `context.GetFlags().antialias_lines`.

Reply:
its probably ok as is